hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-services/hadoop-yarn-services-core/src/main/java/org/apache/hadoop/yarn/service/ServiceScheduler.java

```diff
@@ -1043,9 +1041,7 @@ private boolean terminateServiceIfAllComponentsFinished() {
 
     if (shouldTerminate) {
       LOG.info("All component finished, exiting Service Master... "
-          + ", final status=" + (failedComponents.isEmpty() ?
-          "Succeeded" :
-          "Failed"));
+          + ", final status={}", (failedComponents.isEmpty() ? "Succeeded" : "Failed"));
```

Comment:
I'd prefer the ? : clause on its own line

Reply:
Thanks for your suggestion! I will modify the code.